VolleyManagement.UI/spa/index.html

```diff
@@ -1,14 +1,21 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <title>VolleyManagement.UI</title>
   <base href="/">
 
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <meta name="viewport"
+        content="width=device-width, initial-scale=1">
+  <link rel="icon"
+        type="image/x-icon"
+        href="favicon.ico">
 </head>
+
 <body>
-  <vm-app metadatafile="test-tournament.json"></vm-app>
+  <vm-app></vm-app>
+  <vm-app-integration metadatafile="test-tournament.json"></vm-app-integration>
```

Comment:
I don't think we need it here if this is the main app. But also we might need some local playground to test that

Reply:
If we remove "vm-app" or "vm-app-integration" angular bootstrap mechanism will not handle both cases, because on dev start it will not find root component tag. I put them both to allow us switch between scenarious without any other specific actions